include/class.ticket.php

```diff
@@ -2533,6 +2541,7 @@ function merge($tickets) {
                     $child->getThread()->setExtra($parentThread);
 
                 $child->setMergeType($options['combine']);
+                $child->setState('closed');
```

Comment:
This is problematic - since wording can be different depending on install language. I don't have a solution at hand!